index.html

```diff
@@ -167,6 +167,7 @@ <h2>Contact me</h2>
                     <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"
                         maxlength="500" required></textarea>
                 </li>
+                <span id="emailError">Your email should be in lower case</span>
```

Comment:
- [ ] Good job adding the email validation :smile: , however this alert message should be display only when the email in the form contains uppercase , you should go thought this material and add that feature using JavaScript :+1: 

![Screenshot from 2023-02-03 14-03-01](https://user-images.githubusercontent.com/80458199/216600134-bf832245-4a84-4aa2-ba20-49523500e09c.png)